jhipster-dependencies/pom.xml

```diff
@@ -54,8 +54,8 @@
         <de.flapdoodle.embed.mongo.version>2.2.0</de.flapdoodle.embed.mongo.version>
         <dropwizard-metrics.version>4.1.7</dropwizard-metrics.version>
         <guava.version>29.0-jre</guava.version>
-        <!-- waiting for https://github.com/jhipster/generator-jhipster/issues/11244 -->
-        <hazelcast-hibernate53.version>1.3.2</hazelcast-hibernate53.version>
+        <hazelcast.version>4.0.1</hazelcast.version>
+        <hazelcast-hibernate5.version>2.0.0</hazelcast-hibernate5.version>
```

Comment:
Is this still useful? It doesn't seem to been used

Reply:
Isn't that useful when used as 2nd level cache?

Reply:
I was talking about the maven property: it's not used in the pom.xml

Reply:
It's overriding the version defined in the spring boot parent pom

Reply:
Oh, I see. 
But I'd rather keep an explicit dependency management like what we have today (i.e. a property and an explicit dependency in the `dependencyManagement` section of jhipster-dependencies artifact), it's much easier to maintain and less error prone.
Otherwise you have to know that this property is declared in the parent of the parent, and is overrided here.

Reply:
done